release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -1288,6 +1305,61 @@ You can attempt to resolve the issue with the steps in this link:https://kb.vmwa
 
 * The Console Operator does not properly update the `Ingress` resource with the `componentRoutes` conditions for either of the console's routes (`console` or `downloads`). (link:https://bugzilla.redhat.com/show_bug.cgi?id=1954148[*BZ#1954148*])
 
+* If you are using {sandboxed-containers-first}, you cannot use the `hostPath` volume in a {product-title} cluster to mount a file or directory from the host node’s file system into your pod.
++
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1904609[*BZ#1904609*])
+
+* If you are running Fedora on {sandboxed-containers-first}, you need a workaround to install some packages. Some packages, like `iputils`, require file access permission changes that {product-title} does not grant to containers by default. To run containers that require such special permissions, it is necessary to add an annotation to the YAML file describing the workload, which tells `virtiofsd` to accept such file permissions for that workload. The required annotations are:
++
+[source,yaml]
+----
+io.katacontainers.config.hypervisor.virtio_fs_extra_args: [ "-o", "modcaps=+sys_admin", "-o", "xattr" ]
+----
++
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1915377[*BZ#1915377*])
+
+* In the 4.8 release, adding a value to `kataConfgPoolSelector` using the {product-title} web console results in `scheduling.nodeSelector` to be populated with an empty value. Pods using `RuntimeClass` with the value of `kata` may be scheduled to nodes that do not have the Kata Containers runtime installed.
++
+You can work around this issue by specifying the `nodeSelector` value manually in the `RuntimeClass` `kata` by running the following command:
```

Comment:
```suggestion
To work around this issue, specify the `nodeSelector` value manually in the `RuntimeClass` `kata` by running the following command:
```
just trying to avoid "by [gerund] by [gerund]" here :)